lib/node-modules-paths.js

```diff
@@ -2,7 +2,7 @@ var path = require('path');
 var fs = require('fs');
 var parse = path.parse || require('path-parse');
 
-module.exports = function nodeModulesPaths(start, opts) {
+module.exports = function nodeModulesPaths(x, start, opts) {
```

Comment:
I wasn't sure if it was a publicly exposed function, since it wasn't listed in the doc.

`x` is the request being made (took the variable name from other parts of the code). It can easily be moved to the end of the argument list, for sure

Reply:
Everything reachable is part of the public API; docs are irrelevant - so any break in anything that's exposed would be semver-major.